qt/scientific_interfaces/Indirect/IndirectFitOutputOptionsView.cpp

```diff
@@ -0,0 +1,124 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#include "IndirectFitOutputOptionsView.h"
+
+#include <QMessageBox>
+
+namespace MantidQt {
+namespace CustomInterfaces {
+namespace IDA {
+
+IndirectFitOutputOptionsView::IndirectFitOutputOptionsView(QWidget *parent)
+    : IIndirectFitOutputOptionsView(parent),
+      m_outputOptions(new Ui::IndirectFitOutputOptions) {
```

Comment:
its best to avoid using new wherever possible, maybe `std::make_unique<Ui::IndirectFitOutputOptions>()`